dependencies.props

```diff
@@ -63,7 +63,7 @@
   <!-- Package dependency verification/auto-upgrade configuration. -->
   <PropertyGroup>
     <BaseDotNetBuildInfo>build-info/dotnet/</BaseDotNetBuildInfo>
-    <DependencyBranch>master</DependencyBranch>
+    <DependencyBranch>feature/utf8string</DependencyBranch>
```

Comment:
Same comment as the other 2.  This property is used for a bunch of different repos, and not all these repos have this branch.  (buildtools on line 86, optimization on line 90)